src/Form/Type/TagValueType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Algatux\InfluxDbBundle\Form\Type;
+
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Choice type listing available tag values of a measurement tag key.
+ *
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class TagValueType extends AbstractInfluxChoiceType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $resolver
+            ->setRequired(['measurement', 'tag_key'])
+            ->setDefaults([
+                'choices' => function (Options $options) {
+                    return $this->loadChoicesFromQuery(
+                        sprintf(
+                            'SHOW TAG VALUES FROM "%s" WITH KEY = "%s"',
+                            $options['measurement'],
+                            $options['tag_key']
+                        ),
+                        'value',
+                        $options['connection']
+                    );
+                },
+                'choices_as_values' => true,
+            ])
+            ->setAllowedTypes('measurement', ['string'])
+            ->setAllowedTypes('tag_key', ['string'])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getBlockPrefix()
+    {
+        return 'influx_tag_value';
```

Comment:
I never see the `_type` suffix on any bundles.

It's a block prefix (old name) and we already know it's a type.

Adding this suffix will be cumbersome and this is not recommended.